hal/halx86/include/hal.h

```diff
@@ -36,9 +36,10 @@
 #include <ndk/kefuncs.h>
 #include <ndk/rtlfuncs.h>
 
-/* For MSVC, this is required before using DATA_SEG (used in pcidata) */
+/* Required before using CODE_SEG & DATA_SEG */
 #ifdef _MSC_VER
 # pragma section("INIT", read,execute,discard)
+# pragma section("INITDATA", read, discard)
```

Comment:
Should https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=DATA_SEG%28%22INIT%22%29 be updated too?

Style: space or not, between the attributes?

Reply:
> https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=DATA_SEG%28%22INIT%22%29

These are actually breaking 'build-linux (clang, ...)':
`error: '...' causes a section type conflict with 'HalInitSystem'`

Reply:
Agreed, a follow-up will be fine, then.